src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -458,8 +458,9 @@ function updateScroll() {
   }
 }
 
-watch(() => itemElements.value, () => {
+watch(() => props.items, async () => {
```

Comment:
Sorry, but why is this fix needed?

Reply:
After the updates that watch was not covered by the tests, so after writing some tests for that Anton and I discovered that the watch was not working as it should. We tested manually and found that `itemsElements.value` was not changing and therefore the watch was not being triggered. Watching `props.items.value` instead (and also waiting for `nextTick` before setting the new height) solve the issue.